classes/context/ContextDAO.inc.php

```diff
@@ -144,10 +144,9 @@ function getBySetting($settingName, $settingValue, $contextId = null) {
 	function resequence() {
 		$result = $this->retrieve('SELECT ' . $this->primaryKeyColumn . ' AS context_id FROM ' . $this->tableName . ' ORDER BY seq');
 		$i=1;
-		for ($i=1; $row = (array) $result->current(); $i+=2 && $result->next()) {
+		for ($i = 1; $row = (array) $result->current(); $i += 1 && $result->next()) {
 			$this->update('UPDATE ' . $this->tableName . ' SET seq = ? WHERE ' . $this->primaryKeyColumn . ' = ?', [$i, $row['context_id']]);
-			$result->next();
-			$i+=2;
+			$i += 1;
```

Comment:
I think `$i` is already being incremented in the `for` statement, so could this not be removed?

Reply:
updated - replaced the `for` loop with a `foreach`